### PR TITLE
I'm viroopaksh-utrayak

### PR DESCRIPTION
**Pull request by viroopaksh-utrayak**
## Check List (Check all the applicable boxes) 

- [x] My code follows the code style of this project.
- [ ] My change requires changes to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] All new and existing tests passed.
- [ ] This PR does not contain plagiarized content.
- [ ] The title of my pull request is a short description of the requested changes.

### CODE REVIEW

**Comment by KendallDoesCoding** [comment]
I would like to suggest changes in a bit on this.

**Review by KendallDoesCoding — CHANGES_REQUESTED** [review]

**Review by emmalearnscode — APPROVED** [review]

**Review by kabir0x17 — APPROVED** [review]

File: public/data/viroopaksh-utrayak.json [hunk 1]
```diff
@@ -0,0 +1,33 @@
+{
+  "name": "viroopaksh-utrayak",
+  "type": "personal",
+  "bio": "well I'm a programmer(java) but first of all I'm a learner",
+  "avatar": "https://github.com/viroopaksh-utrayak.png",
+  "links": [
+    {
+      "name": "Here is my Github Accnt",
```
Comment (kabir0x17) [inline public/data/viroopaksh-utrayak.json:8]: If you want to improve this pr then:
Spelling correction
```suggestion
      "name": "Here is my Github Account",
```

File: public/data/viroopaksh-utrayak.json [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+      "url": "https://github.com/viroopaksh-utrayak",
+      "icon": "github"
+    },
+    {
+      "name": "Here is my Twitter Accnt",
```
Comment (kabir0x17) [inline public/data/viroopaksh-utrayak.json:13]:
```suggestion
      "name": "Here is my Twitter Account",
```

File: public/data/viroopaksh-utrayak.json [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+      "url": "https://twitter.com/utrayak",
+      "icon": "twitter"
+    },
+    {
+      "name": "Here is my Instagram Accnt",
```
Comment (kabir0x17) [inline public/data/viroopaksh-utrayak.json:18]:
```suggestion
      "name": "Here is my Instagram Account",
```

File: public/data/viroopaksh-utrayak.json [hunk 4]
```diff
@@ -0,0 +1,33 @@
...
+      "url": "https://www.instagram.com/viroopaksh_utrayak/",
+      "icon": "instagram"
+    },
+    {
+      "name": "Here is my  LinkedIn Accnt",
```
Comment (kabir0x17) [inline public/data/viroopaksh-utrayak.json:23]: You also can do lower-case all these accounts if you want. 
```suggestion
      "name": "Here is my LinkedIn Account",
```

**Review by kabir0x17 — CHANGES_REQUESTED** [review]

File: public/data/viroopaksh-utrayak.json (hunk 1 above)
Comment (KendallDoesCoding) [reply public/data/viroopaksh-utrayak.json:8]: I thought I made a comment asking to do this in my review, its gone now

**Comment by viroopaksh-utrayak** [comment]
hey!!
pl merge my pull request

File: public/data/viroopaksh-utrayak.json (hunk 1 above)
Comment (emmalearnscode) [reply public/data/viroopaksh-utrayak.json:8]: @KendallDoesCoding Maybe the owner would prefer it says Accnt.

File: public/data/viroopaksh-utrayak.json [hunk 5]
```diff
@@ -0,0 +1,33 @@
...
+      "icon": "github"
+    },
+    {
+      "name": "Here is my Twitter Accnt",
+      "url": "https://twitter.com/utrayak",
+      "icon": "twitter"
+    },
+    {
+      "name": "Here is my Instagram Accnt",
+      "url": "https://www.instagram.com/viroopaksh_utrayak/",
```
[the remark this answers is not in the document]
Comment (emmalearnscode) [reply public/data/viroopaksh-utrayak.json:19]: It also doesn't make any difference to the link working or not so it's fine for it to be there

File: public/data/viroopaksh-utrayak.json (hunk 1 above)
Comment (KendallDoesCoding) [reply public/data/viroopaksh-utrayak.json:8]: > @KendallDoesCoding Maybe the owner would prefer it says Accnt.

Yeah, True..

File: public/data/viroopaksh-utrayak.json (hunk 5 above)
[the remark this answers is not in the document]
Comment (KendallDoesCoding) [reply public/data/viroopaksh-utrayak.json:19]: Yeah, but for formatting purposes its better.

File: public/data/viroopaksh-utrayak.json [hunk 6]
```diff
@@ -0,0 +1,33 @@
+{
+  "name": "viroopaksh-utrayak",
+  "type": "personal",
+  "bio": "well I'm a programmer(java) but first of all I'm a learner",
```
[the remark this answers is not in the document]
Comment (KendallDoesCoding) [reply public/data/viroopaksh-utrayak.json:4]: @viroopaksh-utrayak what do you think about this?

**Pull request merged**